### ISSUE TITLE
Fonts

### ISSUE THREAD

**Issue by plakhin** [Opticos/gwsl#9]
Is there a way to use different font for applications UI? I'd like to have same fonts as on Windows (guess Segoe UI is used).

**Comment by Pololot64** [comment]
Yes, it is the same process as changing the font in Linux. Depending on the distro you are using a Google will probably yield helpful results

**Comment by plakhin** [comment]
I'm using Ubuntu 20.04.
Can you help a bit please?
I'm a newbie in Linux with GUI, though been using it many years on servers.
I've added some ttf fonts to ~/.fonts dir and able to use those fonts inside programs, but I also want to change UI fonts.

**Comment by Pololot64** [comment]
I did some digging. After the fonts are installed you can probably add a line like ```gtk-font-name = "DejaVu Sans 11"``` (with your font name) to the ~/.profile file.

**Comment by plakhin** [comment]
Thanks for assistance.
Unfortunately, this doesn't work for me :(
The only app I'm using with gWSL is Sublime Merge, may be it needs some different settings?

**Comment by plakhin** [comment]
Also tried `Text Editor` which seems to be the default app, also no effect.

**Comment by Pololot64** [comment]
Can you post what your .profile file looks like? Or at least the end of it?
